Library/ViewModels/AddNewCardViewModelTests.swift

```diff
@@ -340,41 +335,43 @@ internal final class AddNewCardViewModelTests: TestCase {
 
   func testUnsupportedCardMessage_showsWithInvalidCardBrand_AndExistingCardNumber() {
     self.vm.inputs.viewDidLoad()
+    self.vm.inputs.configure(with: .pledge, project: Project.template)
 
     self.creditCardValidationErrorContainerHidden
       .assertValues([true], "Unsupported card message is hidden on viewDidLoad")
 
-    self.vm.inputs.cardBrand(isValid: false)
-    self.vm.inputs.creditCardChanged(cardDetails: ("123", nil, nil, nil))
+    self.vm.inputs.creditCardChanged(cardDetails: ("123", nil, nil, nil, .generic))
 
     self.creditCardValidationErrorContainerHidden
```

Comment:
Can we also add an assertion for the `unsupportedCardBrandErrorText` output in this test to check that the outputted text is correct for the given intent?

Reply:
Might delete this test as there already is one for it ` func testUnsupportedCardBrandsError_withPledgeIntent()` on `line 439`